### DISCUSSION TITLE
powerwall discharges to zero

### DISCUSSION THREAD

**Discussion by gemu2015** [vloschiavo/powerwall2#69] [Q&A]
i monitor my powerwall2 via iobroker  since i got it in 2018.   in dark winter days i realized that the powerwall completely discharges to zero completely ignoring reserve capacity.  this happens several times every winter. i contacted Tesla several times and they could no explain or repair this behavior.
i now want to know if this could damage my powerwall.  does anybody know if it is possible to read out the internal BMS with cell voltages ?
i guess  this could be a general firmware error.  most users do not monitor the internal api and would never see this.

or does anybody else also have this problem ?

**Comment by emanuelbalea** [comment]
0% isn't actually 0, it's 5% or so. So while its not great it's not too bad. 

Anyway the discharge thing, assuming  you haven't set the reservation to 0%, the reason it goes bellow is because the installer has set a site import limit therefore if load goes above it the powerwall will try its best to supply the difference regardless of reserve. I had the same issue, tho mine set itself after an update. Go through the wizard and don't change anything except the site import limit which you just erase which will disable that function. Note that if your mains wiring can not sustain your loads its best to keep it on unless your sure you're not going to trip anything. Look at your main breaker fuse size and make sure your highest load will not exceed that.

**Comment by fkhera** [comment]
I control reserve using pi
